src/poll/poll.dtos.ts

```diff
@@ -11,9 +11,16 @@ import { ApiProperty, PartialType } from '@nestjs/swagger';
 import { Transform } from 'class-transformer';
 import { ObjectId } from 'mongodb';
 
+export class PollOptionsDto {
+    _id: string;
+    vote_option_name: string;
+    vote_option_emoji: string;
+}
+
 export class CreatePollDto {
 
     @IsMongoId()
+    @IsOptional()
```

Comment:
Am I understanding that the Poll ID is settable here? If so, we can run into duplicate issues? If we auto set, what is the benefit from allowing Poll ID to be set?

Reply:
good point, I think this could be removed. Will update PR